src/procedures/ReserveSecurityToken.ts

```diff
@@ -18,12 +18,23 @@ import { findEvents } from '../utils';
 
 const { bigNumberToDate } = conversionUtils;
 
+/**
+ * - Procedure that reserve the token ticker to an specific owner
```

Comment:
No need for the hyphen

"Procedure that reserves a token ticker (symbol) to be used later on to launch a Security Token"